### ISSUE TITLE
Windows build missing `freetype-config`

### ISSUE THREAD

**Issue by epruesse** [conda-forge/freetype-feedstock#12]
Both `freetype2.pc` and `freetype-config` are missing in the windows builds.

**Comment by djhoese** [comment]
I'm also having issues with this with the aggdraw package. It is making me realize I should probably include more tests in the conda-forge build of aggdraw, but right now it can't build with freetype support because it uses `freetype-config` to find the location of things. See https://github.com/ejeschke/ginga/issues/664 and https://github.com/pytroll/aggdraw/issues/28 for more info.

**Comment by djhoese** [comment]
@ocefpaf It looks like your recent changes to this recipe *should* have added the `freetype-config` binary to this package. Was that your intention? I haven't looked in to it yet, but wanted your opinion on this issue (just skimming through old emails/issues).

**Comment by ocefpaf** [comment]
>  I should probably include more tests in the conda-forge build of aggdraw

That would be nice.

> @ocefpaf It looks like your recent changes to this recipe should have added the freetype-config binary to this package. Was that your intention?

Nope. I'll take a look and see what is happening. Thanks for pinging me.

**Comment by ocefpaf** [comment]
Quick update:

- I don't think we ever have `freetype-config` on Windows, just checked that in the old packages.

- @djhoese your issues may be related to https://github.com/conda-forge/conda-forge.github.io/issues/621

**Comment by djhoese** [comment]
I don't know enough about freetype's config process or building C code on Windows, but we need the equivalent of https://github.com/conda-forge/freetype-feedstock/blob/master/recipe/build.sh#L8 I'm guessing. It seems someone knew it wouldn't work when they wrote the recipe: https://github.com/conda-forge/freetype-feedstock/blob/master/recipe/meta.yaml#L35

**Comment by ocefpaf** [comment]
The build on Windows uses cmake and this is the closest that I could find:

https://github.com/source-foundry/freetype2/commit/c80620c7fe1c11a4a6080cdec10c09ccd6a4edda

**Comment by djhoese** [comment]
Looks like the `.cmake` part of that commit is included in the currently built version used by conda-forge. I don't know enough about cmake to know what to do with it though:

https://ci.appveyor.com/project/conda-forge/freetype-feedstock/branch/master/job/9q3bvsavi7qjqo7a#L870

**Comment by ocefpaf** [comment]
I don't think that creates the `freetype-config` binary, just a "freetype-config" like config info for cmake related projects. (I'm cmake expert either.)

**Comment by djhoese** [comment]
Right, I'm just saying that the commit you linked to doesn't look like it adds anything related to freetype-config that shouldn't be available already. While I was searching around I found this though:

```
    - Use  of  the `freetype-config'  script  to  get compilation  and
      linking  options   is  deprecated   since  it   doesn't  support
      cross-compiling, among other  deficiencies.  Instead, you should
      use the `pkg-config' interface.

      The `configure'  script no longer installs  `freetype-config' by
      default.  For  backwards compatibility,  a new  configure option
      `--enable-freetype-config'   is  provided   that  reverts   this
      decision.
```

from the README here: https://sourceforge.net/projects/freetype/files/freetype2/2.9.1/

**Comment by ocefpaf** [comment]
> Right, I'm just saying that the commit you linked to doesn't look like it adds anything related to freetype-config that shouldn't be available already. 

Yep. Sorry I was not clear on that.

**Comment by djhoese** [comment]
As seen in the above mentioned PR, @ocefpaf do you know of any better way in `setup.py` to find a libraries prefix (to then find its include and lib dirs)?

**Comment by djhoese** [comment]
@ocefpaf Given the message above in the freetype README I decided to add a workaround to my aggdraw package where I use ctype's `find_library` and locate the include dirs from there. I've had users confirm that the conda-forge package for `aggdraw` now properly links to freetype. So I'm not as concerned about this anymore, but I'm not the OP either.

**Comment by ocefpaf** [comment]
Sorry. I did not get back to you on that. I don't really know the best approach to find things on Windows nor if we can build the `freetype-config` unless we use `msys2`.

**Comment by djhoese** [comment]
I'm not sure what `msys2` is, but it sounds like freetype-config is deprecated by the freetype project anyway so in my opinion I think this can be closed as a `wontfix`.

**Comment by ocefpaf** [comment]
> I'm not sure what msys2 is

Sorry. I should've been more specific. Msys2 allows for a `configure/make/make install` build on Windows.
